clippy_lints/src/types.rs

```diff
@@ -1714,6 +1725,18 @@ fn get_numeric_literal<'e>(expr: &'e Expr<'e>) -> Option<&'e Lit> {
     }
 }
 
+fn is_hir_ty_cfg_dependant(cx: &LateContext<'_>, ty: &hir::Ty<'_>) -> bool {
```

Comment:
Could you move this function to `utils/mod.rs`? This seems general enough to be reusable.